### ISSUE TITLE
Unable to access opensearch-dashboards using domain name 

### ISSUE THREAD

**Issue by shubhikaushal7290** [opensearch-project/OpenSearch-Dashboards#3037]
I have setup of opensearch-dashboards, I'm able to view applications logs in opensearch-dashboards while access dashboard using http://ip:PORT/

However, when I add context path/basepath details in values.yaml file of opensearch-dashboard helm charts, then it starts throwing error in opensearch-dashboards pod logs, like mentioned below:

pod logs:

{"type":"log","@timestamp":"2022-12-07T07:09:58Z","tags":["error","opensearch","data"],"pid":1,"message":"[ResponseError]: Response Error"}

values.yaml file detail:

config:
opensearch_dashboards.yml: |
server:
name: dashboards
host: "0.0.0.0"
basePath: "/opensearch-dashboards"
rewriteBasePath: true

<img width="770" alt="domain_name_opensearch_dashboard" src="https://user-images.githubusercontent.com/89378956/206163243-33881239-a9fc-45f2-8db8-71e3e7f9427b.PNG">

<img width="942" alt="opensearch-dashboard logs" src="https://user-images.githubusercontent.com/89378956/206163254-3f5b09a4-483b-49dc-9082-ea15a0a868d3.PNG">

<img width="343" alt="opensearch-dashboard-config" src="https://user-images.githubusercontent.com/89378956/206163265-a2e5011a-a9cf-43c9-ae20-6434d09c636c.PNG">

****** Please help me on this and let me know if require more information.

**Comment by ashwin-pc** [comment]
This might be a something for the https://github.com/opensearch-project/helm-charts repo. 

cc: @opensearch-project/transfer-request

**Comment by kavilla** [comment]
I agree @ashwin-pc, looks like something they should be able to assist with.
